streams/src/test/java/org/apache/kafka/streams/processor/internals/AssignedStreamsTasksTest.java

```diff
@@ -256,9 +256,10 @@ public void shouldResumeMatchingSuspendedTasks() {
     }
 
     @Test
-    public void shouldCloseTaskOnResumeIfTaskMigratedException() {
+    public void shouldCloseTaskOnResumeSuspendedIfTaskMigratedException() {
         mockRunningTaskSuspension();
         t1.resume();
+        t1.initializeTopology();
```

Comment:
`t1.resume()` doesn't do anything. Should we remove it and rename the test? `shouldCloseTaskOnInitializeTopologyIfTaskMigratedException` ?

Reply:
In `assignedTasks.maybeResumeSuspendedTask` we still call both functions, so to make it more general in case we would have some logic inside resume in the future, I still keep it here and leave the title not changed in order to reflect the more general case.

Reply:
Ack. Fine with me.